<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coreDumpUtilFrame</class>
 <widget class="QMainWindow" name="coreDumpUtilFrame">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>380</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>FreeSans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>XPlatform CoreDump Analysis Utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbanner">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>     Cross Platform CoreDump Analysis Utility</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbannerImage">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>-10</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>LogoSJ.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lArchitecture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Architecture</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboArchType">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>77</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>select the architecture</string>
    </property>
    <item>
     <property name="text">
      <string>    MIPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>    X86</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lSysRoot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>SysRoot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSysRootBrowse">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>Give the Path of fileSystem  - fs in the path staging of the sandbox.</string>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSysRootFilePath">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No File Selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApplicationExe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Application-Exe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bApplicationBrowser">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>Select the Application - Exe . The main binary - process.</string>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApplicationFilePath">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No File Selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCoreFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Core-file </string>
    </property>
   </widget>
   <widget class="QPushButton" name="lCoreFileBrowser">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>Give the path of core file - this can be even compressed form like lzo.</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string>  Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCorefileBrowser">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No File Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgetBackTrace">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This will get the back trace, the backtrace log is stored in bt.txt</string>
    </property>
    <property name="text">
     <string>Get BackTrace</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboArchType</tabstop>
  <tabstop>bSysRootBrowse</tabstop>
  <tabstop>bApplicationBrowser</tabstop>
  <tabstop>lCoreFileBrowser</tabstop>
  <tabstop>bgetBackTrace</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
